dtlpylidar/parsers/base_parser.py

```diff
@@ -11,7 +11,7 @@
 logger = logging.Logger(name="file_mapping_parser")
 
 
-class LidarFileMappingParser(dl.BaseServiceRunner):
```

Comment:
Why did you remove the BaseServiceRunner?
it was developed to work as faas as well